src/common/files.ts

```diff
@@ -29,3 +32,45 @@ export function enrichFilePayloads(
     return file
   })
 }
+
+function calculateStringSize(str: string): number {
+  return new TextEncoder().encode(str).length
+}
+
+export function checkRequestBodySize(reqBody: any): void {
```

Comment:
Can we make it type safe and use type not any. We do not wanna use any ever only in some special situations